### PR TITLE
TestClusterTopology: support tiflash in tidb

### LINKED ISSUE #98: Support TiFlash in TiDB
## Development Task
I hope to integrate TiFlash into tidb when it is deployed

### PR DESCRIPTION
**Pull request by CadmusJiang**
### What problem does this PR solve?

Issue Number: close #98  

Problem Summary: support tiflash component in tidb

### What is changed and how it works?

Proposal: [xxx](url) 

What's Changed:
How it Works:
1.build new image(cadmusjiang/base-image), this image installed **tzdata** because tiflash need
2.add TiFlashSpec in testclustertopology.go
3.modify /pkg/tiup/deploy.go/BuildSpecification, add tiflash config
4.init resources expose tiflash port

### Related changes

- Need to cherry-pick to the release branch

### Check List 

Tests 

- Manual test (add detailed scripts or steps below)
1.deploy tidb with tiflash

Side effects

- Breaking backward compatibility

### CODE REVIEW

**Comment by CadmusJiang** [comment]
`numaNode: "0"` is not supported because of permissions.
企业微信截图_8134868d-4c6d-4d11-ac42-3079cf73c0af
image

**Comment by CadmusJiang** [comment]
Other is ok.

File: pkg/tiup/common.go [hunk 1]
```diff
@@ -11,7 +11,7 @@ import (
 const (
 	// Our controller node and worker nodes share the same insecure_key path
 	InsecureKeyPath = "/root/insecure_key"
-	ContainerImage  = "docker.io/mahjonp/base-image:latest"
+	ContainerImage  = "docker.io/cadmusjiang/base-image:latest"
```
Comment (mahjonp) [inline pkg/tiup/common.go:14]:
```suggestion
	ContainerImage  = "docker.io/pingcapqe/base-image:latest"
```

File: api/v1/testclustertopology_types.go [hunk 2]
```diff
@@ -129,6 +129,34 @@ type PrometheusSpec struct {
...
+	// +optional
+	ProxyStatusPort int `json:"proxyStatusPort,omitempty"`
+	// +optional
+	MetricsPort int `json:"metricsPort,omitempty"`
+
+	DeployDir string `json:"deployDir"`
+	DataDir   string `json:"dataDir"`
+
+	// +optional
+	NumaNode string `json:"numaNode,omitempty"`
```
Comment (mahjonp) [inline api/v1/testclustertopology_types.go:151]: Drop it.

**Review by mahjonp — APPROVED** [review]
Rest LGTM

**Comment by mahjonp** [comment]
> `numaNode: "0"` is not supported because of permissions.
> 企业微信截图_8134868d-4c6d-4d11-ac42-3079cf73c0af
> image

On container environment, it's very seldoooooom to use numactl, and we plan to support NUMA through other method. So could you drop the NumaNode field on TiFlashSpec?

**Review by mahjonp — APPROVED** [review]
LGTM

**Pull request merged**